src/subset.rs

```diff
@@ -295,15 +301,16 @@ impl<N: NodeIdT + Rand> Subset<N> {
         self.ba_results.values().filter(|v| **v).count()
     }
 
-    fn try_binary_agreement_completion(&mut self) -> Option<BTreeMap<N, ProposedValue>> {
+    fn try_binary_agreement_completion(&mut self, proposer_id: &N) -> Option<SubsetOutput<N>> {
         if self.decided || self.count_true() < self.netinfo.num_correct() {
             return None;
         }
         // Once all instances of BA have completed, let C ⊂ [1..N] be
         // the indexes of each BA that delivered 1. Wait for the output
         // v_j for each RBC_j such that j∈C. Finally output ∪ j∈C v_j.
         if self.ba_results.len() < self.netinfo.num_nodes() {
-            return None;
+            return Some(SubsetOutput::Contribution(proposer_id.clone(),
+             self.broadcast_results.get(proposer_id).map(|x|x.clone()).expect("internal error: already inserted entry not found")));
```

Comment:
`rustfmt` won't accept this in Travis. Please check the error message.

Reply:
@vkomenda Fixed